cpp/include/cudf/utilities/device.hpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+
+namespace cudf {
+namespace experimental {
+
+/**
+ * @brief Returns the version number of the current CUDA Runtime instance.
+ * The version is returned as (1000 major + 10 minor). For example,
+ * CUDA 9.2 would be represented by 9020.
+ *
+ * This function returns -1 if runtime version is NULL.
+ *
+ * @return Integer containing the version of current CUDA Runtime.
+ */
+int get_cuda_runtime_version();
```

Comment:
One change I would do is specify `void` in the list of parameters rather than empty list `()` in the signature of the function _declaration_ (definition shouldn't matter). In C++ it shouldn't matter (and this is obviously a C++ header).

_But_ given the `cython` interaction, the concern is that cython may translate C++ headers into C headers. If that happens, then empty parameter list of a C-function _declaration_ does _not_ mean `void`. It means unspecified number of arguments (or, unknown at the time of declaration). See, for example, here: https://softwareengineering.stackexchange.com/a/287002 . 

That _can_ be a potential source of ambiguity that can lead to very hard to catch bugs. Specifying `void` instead of `()` should eliminate that possibility.

Reply:
Noted Andrei. Will keep this in mind in future PRs but since I removed this file can you re-review the changes I performed? The api's are now in cython directly calling the CUDA apis.